docs/reference/koswat_surroundings.md

```diff
@@ -0,0 +1,88 @@
+# Surroundings
+
+In koswat's real domain, a dike is surrounded by all sort of surroundings, these could be nature or human-built structures. During our analyisis we make use of (`.shp`) maps where a dike's traject is shown together with all the human-built structures in its proximity. 
+
+We categorize these structures in the following two types:
+
+- [Obstacles](#obstacles), surroundings that __cannot__ be modified in any way.
+- [Infrastructures](#infrastructures), as opposite to the obstacles, surroundings that can be repaired or removed and added again (elsewhere).
+
+
+|![Koswat dike surroundings representation](./imgs/dike_traject_with_surroundings.png)|
+|:--:|
+|Image 1. Surroundings representation|
+
+To define said surroundings we make use of `.csv` files, each file for a specific type of surrounding that needs to be "enabled" in the `koswat_general.ini` settings file.
+
+In addition, surroundings can be defined for the inside or the outside of a dike. At the current `Koswat` version (unreleased `v0.15.0`) we __only support__ surroundings in the inside of a dike ( `binnendijks`).
+
+## Surroundings `.csv` files
+
+The surroundings files follow all the same format both in naming and content. 
+
+Their file name has the form of `T{dike_traject}_{type_of_surrounding}.csv`:
+- The `{dike_traject}` is the dike's traject code name, usually something like `10_1`,
+- the `{type_of_surrounding}` is the name of surrounding that will be imported (check further in the corresponding subsections of [obstacles](#obstacle-types)).
+
+Their headers are divided in the following columns:
+
+- `SECTIE`, section of the location in the dike's traject,
+- `Xcoord`, x coordinate of the location,
+- `Ycoord`, y coordinate of the location, 
+- `afst_{x}m`, distance from the location, where 'x' are all the values between `5` and `200` with steps of 5, thus 40 columns. The values in these cells are called define whether there are obstacles at this distance or not and in `Koswat` are usually built in the `SurroundingsPoint.surroundings_matrix`, each type of surrounding interprets the weights in a different manner.
+
+
+## Obstacles
+
+An obstacle usually represents a building, because they cannot be removed they become a constraint to the possible reinforcements that can be applied at a given location. For instance, if a dike's reinforcement will become 5 meters wider, but within that distance there are obstacles, said reinforcement will be discarded as a possible option at that location.
```

Comment:
Not only buildings, right? Also railways or waters.

Reply:
Yes, you are right, but that's why I say 'usually' instead of 'always', furthermore below you may find the subtypes being mapped.